### PR TITLE
Add keymap for soda/mango

### PR DESCRIPTION
**Pull request by gezhaoyou**
## Description
Add keymap for soda/mango

## QMK Pull Request 
https://github.com/qmk/qmk_firmware/pull/21100#event-9438160621
Support rgb matrix:
https://github.com/qmk/qmk_firmware/pull/21385#event-9759338521

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/soda/mango/mango.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "name": "MF17",
+    "vendorId": "0x3A54",
+    "productId": "0x4F5D",
+    "keycodes": ["qmk_lighting"],
+    "menus": ["qmk_rgb_matrix"],
```
Comment (Xelus22) [inline v3/soda/mango/mango.json:6]: Where is the RGB code in QMK?

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/soda/mango/mango.json (hunk 1 above)
Comment (gezhaoyou) [reply v3/soda/mango/mango.json:6]: I have pr rgb matrix code to qmk

Comment (gezhaoyou) [reply v3/soda/mango/mango.json:6]: see https://github.com/qmk/qmk_firmware/pull/21385#event-9759338521

File: v3/soda/mango/mango.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+    "name": "MF17",
```
Comment (Xelus22) [inline v3/soda/mango/mango.json:2]: MF17? Or should it be renamed to Mango?

Comment (gezhaoyou) [reply v3/soda/mango/mango.json:2]: I confirm that my product model is MF17, so MF17 is correct

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**